db/db.ts

```diff
@@ -78,9 +78,32 @@ async function storeSavedFileMetadata(file: StoredFile): Promise<string | null>
     return storedFile.insertedId.toString()
 }
 
+async function getFileById(id: string): Promise<StoredFile | null> {
```

Comment:
I would sugest to change the name of the function to `getUploadMetadataById`

Reply:
Done